src/components/Widget/Widget.tsx

```diff
@@ -33,18 +33,24 @@ interface WidgetProps extends WidgetOwnProps {
     loading?: boolean,
     parentCursor?: string,
     customWidgets?: ObjectMap<CustomWidget>,
+    skipWrappingWidgets?: string[],
     showWidget: boolean,
     rowMetaExists: boolean,
     dataExists: boolean
 }
 
 const skeletonParams = { rows: 5 }
 
+const SKIP_WRAPPING_WIDGETS: string[] = [WidgetTypes.AssocListPopup, WidgetTypes.PickListPopup]
+
 export const Widget: FunctionComponent<WidgetProps> = (props) => {
     if (!props.showWidget) {
         return null
     }
-    if (props.meta.type === WidgetTypes.AssocListPopup || props.meta.type === WidgetTypes.PickListPopup) {
+
+    const skipCardWrapping = SKIP_WRAPPING_WIDGETS.includes(props.meta.type) && props.skipWrappingWidgets?.includes(props.meta.type)
```

Comment:
Why conjunction has appeared here instead of disjunction?

Reply:
fixed